watchMeTests/openImageInGallary.test.js

```diff
@@ -0,0 +1,27 @@
+const {snippetHomepageOpened} = require('./common');
+const suitest = require('suitest-js-api');
+const {assert, VRC, PROP, COMP, VALUE} = suitest;
+const {snapshotElement} = require('./utils');
+const {imageCandiesFocused, gallery, imageCandiesInGalleryRepo} = require('./elements');
+
+describe('Opening Image in Gallery', async() => {
```

Comment:
The idea of describe/it is that it should be human readable, i.e. "Describe image gallery. It should open correct image".
Please, look through test files and update in such fashion.